test/hotel_manager_test.rb

```diff
@@ -0,0 +1,183 @@
+require 'test_helper.rb'
```

Comment:
This line was causing the tests to be unable to run. I had to change it to look like the other tests.
```suggestion
require_relative 'test_helper'
```